app/assets/stylesheets/components/browse.scss

```diff
@@ -8,18 +8,28 @@
     @include only-mobile {
       width: 87%;
     }
+
+    .resources-index & {
+      max-width: none;
+  
+      @include mobile-container;
+    }
   }
 
   @include e(tabs-items) {
-    margin: 0;
-    padding: 0;
+    @include list;
   }
 
   @include e(tabs-item) {
-    list-style-type: none;
+    @include list-item--row(0);
+    // For browse__tab:hover to inherit and overwrite default hover styling
+    color: $white;
+
+    display: inline-block;
   }
 
   @include e(tab) {
+    @include link-reset;
     appearance: none;
```

Comment:
Perhaps we could update the button-reset mixin and use it here instead? or combine the two somehow

Reply:
I did wonder about that. I chose to keep this separate as the color inherit styling could lead to unwanted behaviour, so wouldn't want to apply that to lots of buttons. If that makes sense?

Reply:
In theory, we shouldn't be using the default button mixin, and only ever the specific button we need (which is what applies the colour), so I think we can get away with using a color inherit/currentColor property here

Reply:
We could also have link-reset and button-reset, and use link-reset within button-reset?

Reply:
The only thing is that it inherits from the parent. So for example:

```
<div style="color: black">
  <button class="button">button</button>
</div>


.button {
  @include link-reset;
  color: red;
}
```

In this scenario on hover the button text will go black, but the desired effect is that it remains red. So to achieve this the button would need a wrapper with color: red. So it's quite hacky and could lead to buttons actually getting hover colors that differ from the normal color of the button in a non-consistent way.

Reply:
Okay, let's leave as is for now